### PR TITLE
Writes out lists for roles in the Dynamic Rulesets Code

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

Alphabetizes and spans out the lists for all of the restricted_roles and protected_roles in the three different DM files for the Dynamic rulesets. @Timberpoes in #63960 already addresses this for Latejoin Provocateur and Roundstart Revolutionary and Cultist, so those are untouched.

## Why It's Good For The Game

I was promised cookie :). Also, this will apparently help maints review PRs without having stuff fall through the racks (like what #63960 fixes).

## Changelog

:cl:
code: Nanotrasen has implemented improved checking in their protocols, only delaying the inevitability that the AI can one day get to be a blood cultist.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -408,7 +454,7 @@
 
 //////////////////////////////////////////////
 //                                          //
-//               REVS                 //
+//               REVS                       //
```
Comment (san7890) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:457]: i couldn't help myself

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 2]
```diff
@@ -334,7 +377,10 @@
 	antag_datum = /datum/antagonist/nukeop
 	var/datum/antagonist/antag_leader_datum = /datum/antagonist/nukeop/leader
 	minimum_required_age = 14
-	restricted_roles = list(JOB_CAPTAIN, JOB_HEAD_OF_SECURITY) // Just to be sure that a nukie getting picked won't ever imply a Captain or HoS not getting drafted
+	restricted_roles = list(
+		JOB_CAPTAIN,
+		JOB_HEAD_OF_SECURITY,
+	) // Just to be sure that a nukie getting picked won't ever imply a Captain or HoS not getting drafted
```
Comment (san7890) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:383]: i'm not sure if the comment on this line is misplaced, but i think it makes sense

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm (hunk 1 above)
Comment (Mothblocks) [reply code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:457]: Don't I hate all of these comments and want them dead

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm (hunk 2 above)
Comment (Mothblocks) [reply code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:383]: i would've put it on 379 but it doesn't matter

File: code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm [hunk 3]
```diff
@@ -50,8 +50,18 @@
...
+		JOB_CAPTAIN,
+		JOB_DETECTIVE,
+		JOB_HEAD_OF_PERSONNEL,
+		JOB_HEAD_OF_SECURITY,
+		JOB_SECURITY_OFFICER,
+		JOB_WARDEN,
+	)
+	restricted_roles = list(
+		JOB_AI,
+		JOB_CYBORG
```
Comment (Mothblocks) [inline code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm:63]:
```suggestion
		JOB_CYBORG,
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]
looksg ood

File: code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm (hunk 3 above)
Comment (san7890) [reply code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm:63]: okf ixed

**Review by Timberpoes — APPROVED** [review]
Simple code cleanup. No point holding this to the 24h mark as it's purely code formatting improvements.

One cookie as promised.

image

**Comment by san7890** [comment]
> Simple code cleanup. No point holding this to the 24h mark as it's purely code formatting improvements.
> 
> 
> 
> One cookie as promised.
> 
> 
> 
> image
> 
> 

Let's GOOOOOOOOO

**Pull request merged**